### PR TITLE
Check that there are checks.

### PR DESCRIPTION
**Pull request by AceFire6**
Closes: #10

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by zoidbergwill** [comment]
Maybe we should make it return a message of some sort when no checks are done.

File: healthcheck/contrib/django/status_endpoint/tests/test_views.py [hunk 1]
```diff
@@ -69,13 +69,18 @@ def test_dont_check_files(self):
...
-        response_json = json.loads(response.content)
-        self.assertTrue(
-            "quiesce file doesn't exist" not in response_json)
-        self.assertTrue(
-            'Django Databases Health Check' not in response_json)
-        self.assertEqual(response.status_code, 500)
+        response = {
+            'content': json.loads(response.content),
+            'status': response.status_code,
+            }
```
Comment (snitch) [inline healthcheck/contrib/django/status_endpoint/tests/test_views.py]: E123 closing bracket does not match indentation of opening bracket's line

File: healthcheck/contrib/django/status_endpoint/tests/test_views.py [hunk 2]
```diff
@@ -69,13 +69,18 @@ def test_dont_check_files(self):
...
+
+        expected_response = {
+            'content': 'There were no checks.',
+            'status': 200,
+        }
+
+        self.assertEqual(response, expected_response)
+
```
Comment (snitch) [inline healthcheck/contrib/django/status_endpoint/tests/test_views.py]: W391 blank line at end of file

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: healthcheck/healthcheck.py [hunk 3]
```diff
@@ -173,9 +173,10 @@ def _assess_overall_status(self):
         failed_critical_checks = [
             check for check in failed_checks if check.is_critical]
 
-        # If any of critical checks failed, or all checks failed, system state
-        # is False (means "bad").
-        if failed_critical_checks or len(failed_checks) == len(self._checks):
+        # If any of critical checks failed, or all checks failed, system
+        # state is False (means "bad").
+        if self._checks and (failed_critical_checks
```
Comment (ekoval) [inline healthcheck/healthcheck.py]: I recomment to move this new condition to the very beginning of the method:
```
if not checks:
    return True
```

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by ekoval** [comment]
:+1:

**Pull request merged**